src/components/MainCalendarCanvas/MainCalendarCanvas.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+  HOURS_IN_DAY,
+  MAIN_CALENDAR_TIMES,
+  WEEK_LENGTH,
+} from '../../constants';
+import EventList from '../EventList/EventList';
+import styles from './MainCalendarCanvas.module.scss';
+
+const MainCalendarCanvas = () => {
+  const mainCalendarCanvas = [];
+  for (let i = 0; i < WEEK_LENGTH; i++) {
```

Comment:
Also, something like this could work too:
```
const MainCalendar = () =>  <>{Array.from({ length: WEEK_LENGTH }, (_, i) => 
  Array.from({ length: HOURS_IN_DAY + 1 }, (_, j) => (
    <div key={`${i}-${j}`} className={styles.cell}></div>
  )))}
  </>
```